### PR TITLE
Added no-async-without-await converter and unit tests

### LINKED ISSUE #165: Missing converter: no-async-without-await
### 💥 Missing Converter

-   `tslint-to-eslint-config` version: 1.2.0
-   `typescript-eslint` version: 2.3.2

#### TSLint Rule Name

`no-async-without-await`

### PR DESCRIPTION
**Pull request by asponda**
## PR Checklist

-   [X] Addresses an existing issue: fixes #165 
-   [X] That issue was marked as [`status: accepting prs`](https://github.com/typescript-eslint/tslint-to-eslint-config/labels/status%3A%20accepting%20prs)

## Overview

Added converter for TSLint rule "no-async-without-await" to ESLint rule "require-await" and unit tests

### CODE REVIEW

File: src/rules/converters/no-async-without-await.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { RuleConverter } from "../converter";
+
+export const convertNoAsyncWithoutAwait: RuleConverter = () => {
+    return {
+        rules: [
+            {
+                ruleName: "require-await",
```
Comment (JoshuaKGoldberg) [inline src/rules/converters/no-async-without-await.ts]: Wait, I just noticed - this should be:

```suggestion
                ruleName: "@typescript-eslint/require-await",
```

Comment (asponda) [reply src/rules/converters/no-async-without-await.ts]: Hi, I develop the suggested modification and pull new commits, but the tests end-to-end has failed :(

**Comment by JoshuaKGoldberg** [comment]
I'll take a look at the end to end tests failures...

**Comment by JoshuaKGoldberg** [comment]
Ignoring end-to-end test failures as they're removed in `master`. 🤷‍♂

**Pull request merged**